### PR TITLE
Add module gzrt (#6918)

### LINKED ISSUE #6918: new module: gzrt aka gzrecover
I'd like to add a module that wraps this: https://github.com/arenn/gzrt

gzrecover is a program that will attempt to extract any readable data
out of a gzip file that has been corrupted.

### PR DESCRIPTION
**Pull request by mazzalab**
Added a new module

## PR checklist

Closes #6918

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [x] `nf-core modules test <MODULE> --profile singularity`
    - [x] `nf-core modules test <MODULE> --profile conda`

### CODE REVIEW

**Review by LouisLeNezet — CHANGES_REQUESTED** [review]
Just a few changes needed before merging it ;)

**Comment by mazzalab** [comment]
> Just a few changes needed before merging it ;)

I couldn't pipe gzrecover to gzip because the former specifies the output file name with a `-o` params. At least, I did not know what to do. Thus, I've resorted to a named pipe that should prevent me from allocating temp space.

Now I have got an odd error with conda checks. I'm waiting for feedback and we should be done

**Review by LouisLeNezet — APPROVED** [review]
Seems good to me 😄

**Pull request merged**
